### PR TITLE
Updated drought in user guide & quickstart

### PR DESCRIPTION
**Pull request by jasSust**
- Added additional text for drought in user guide
- Updated table values for drought in Quickstart

### CODE REVIEW

**Review by bcwaldon — CHANGES_REQUESTED** [review]

File: jekyll/userguide.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ We also use fire weather as an indicator of fire risk.  This indicator shows the
...
-
+To estimate an asset’s exposure to drought, we utilized a drought index based on CMIP6 simulations of precipitation and temperature, from which we estimate potential evapotranspiration. This is the more suitable index for agricultural assets like farms and farm production sites. The drought index, also referred to as the standardized precipitation evapotranspiration index [SPEI] (https://spei.csic.es/home.html), represents the magnitude of precipitation deficits (negative magnitude) or surplus (positive magnitude) along with accounting for evaporation over the preceding 12-month period. Values between -1.5 and -2.0 indicate moderate dryness, while values below -2.0 indicate severe dryness. High risk, exceptional droughts fall below -2.0 on the standard precipitation evapotranspiration index (SPEI). Drought index values are calculated relative to a historical 1950-2014 baseline period, and future droughts can reach index values below -3.0 if the droughts are unprecedented relative to that 1950-2014 historical period.
```
[the remark this answers is not in the document]
Comment (bcwaldon) [reply jekyll/userguide.md:36]: Note that the link is broken in the rendered HTML

[the remark this answers is not in the document]
Comment (jasSust) [reply jekyll/userguide.md:36]: @bcwaldon No change, this should be reverting back to what we had before.

[the remark this answers is not in the document]
Comment (jasSust) [reply jekyll/userguide.md:36]: The link appears to be working on my end. Where are you seeing that?

File: docs/userguide.html [hunk 2]
```diff
@@ -83,8 +83,7 @@ <h4 id="wildfire-susceptibility">Wildfire Susceptibility</h4>
...
-
+<p>To estimate an asset’s exposure to drought, we utilized a drought index based on CMIP6 simulations of precipitation and temperature, from which we estimate potential evapotranspiration. This is the more suitable index for agricultural assets like farms and farm production sites. The drought index, also referred to as the standardized precipitation evapotranspiration index [SPEI] (https://spei.csic.es/home.html), represents the magnitude of precipitation deficits (negative magnitude) or surplus (positive magnitude) along with accounting for evaporation over the preceding 12-month period. Values between -1.5 and -2.0 indicate moderate dryness, while values below -2.0 indicate severe dryness. High risk, exceptional droughts fall below -2.0 on the standard precipitation evapotranspiration index (SPEI). Drought index values are calculated relative to a historical 1950-2014 baseline period, and future droughts can reach index values below -3.0 if the droughts are unprecedented relative to that 1950-2014 historical period.</p>
```
Comment (bcwaldon) [inline docs/userguide.html:86]: @jasSust you can see that the SPEI link is no longer rendering properly in HTML. Please revert the corresponding change on line 36 in userguide.md

**Review by bcwaldon — CHANGES_REQUESTED** [review]

File: docs/userguide.html (hunk 2 above)
Comment (jasSust) [reply docs/userguide.html:86]: @bcwaldon Ahh, got it -- should be fixed, now!

**Pull request merged**